src/catch2/internal/catch_run_context.cpp

```diff
@@ -447,6 +449,7 @@ namespace Catch {
         AssertionResult result(m_lastAssertionInfo, CATCH_MOVE(tempResult));
 
         assertionEnded(CATCH_MOVE(result) );
+        resetAssertionInfo();
```

Comment:
Is this actually neeeded? `RunContext::assertionEnded` (called on the line before this one) already calls `resetAssertionInfo`.

Reply:
Wait sorry, I didn't notice that you changed `assertionEnded` to not reset assertion info anymore.